style.css

```diff
@@ -33,13 +33,18 @@ html, body {
     text-align: center;
 }
 
+#pbrMath, #inputs, .dg {
```

Comment:
What is `.dg`?  I don't see it anywhere else.

Reply:
`.dg` is control panel
> [libs\dat.gui.min.js](https://github.com/KhronosGroup/glTF-WebGL-PBR/blob/master/libs/dat.gui.min.js)


![dg](https://user-images.githubusercontent.com/7625588/34023486-2c5619dc-e180-11e7-9810-a2a24aece2da.png)